# This message contains GLONASS ephemeris data

uint32 sat
GnssTimeMsg ttr
GnssTimeMsg toe
int32  freqo
uint32 iode
uint32 health
uint32 age
float64 ura
float64 pos_x
float64 pos_y
float64 pos_z
float64 vel_x
float64 vel_y
float64 vel_z
float64 acc_x
float64 acc_y
float64 acc_z
float64 tau_n
float64 gamma
float64 delta_tau_n